src/sentry/db/models/fields/node.py

```diff
@@ -218,7 +213,13 @@ def to_python(self, value):
             # to load data from, and no data to save.
             value = None
 
-        return NodeData(self, node_id, value, wrapper=self.wrapper)
+        return NodeData(
```

Comment:
Do you plan to keep the NodeField abstraction at all ? I think that was specific to the Django ORM

Reply:
Since Event will not be a Django model long term, we won't be able to keep NodeField there. This code will have to stay however since it will still be used by RawEvent.

Reply:
Do you know why RawEvent uses this at all ? In a world where there is no postgres event anymore, it seems wrong to keep a custom postgres field for the event data.

Reply:
Well RawEvent is just an event that failed processing and can be retried, so I guess we need to store the event body somewhere even though it's not actually saved yet. As far as I know this is really low volume and there is no plan to move this out of Postgres. Since it is a Postgres model, it seems ok that it has a custom postgres field to hold the relevant node ID?